deploy/chart/templates/grafanaDashboard.yaml

```diff
@@ -0,0 +1,2346 @@
+            "overrides": []
+        },
+        "gridPos": {
+            "h": 9,
+            "w": 12,
+            "x": 0,
+            "y": 19
+        },
+        "id": 15,
+        "options": {
+            "legend": {
+            "calcs": [
+                "last"
+            ],
+            "displayMode": "table",
+            "placement": "right"
+            },
+            "tooltip": {
+            "mode": "multi"
+            }
+        },
+        "targets": [
+            {
+            "exemplar": true,
+            "expr": "capi_machinedeployment_status_replicas_unavailable{exported_namespace=~\"$namespace\", machinedeployment=~\"$cluster.*\"}",
+            "interval": "",
+            "legendFormat": "{{ "{{" }}machinedeployment{{ "}}" }}: unavailable",
+            "refId": "A"
+            },
+            {
+            "exemplar": true,
+            "expr": "capi_machinedeployment_spec_replicas{exported_namespace=~\"$namespace\", machinedeployment=~\"$cluster.*\"}",
+            "hide": false,
+            "interval": "",
+            "legendFormat": "{{ "{{" }}machinedeployment{{ "}}" }}: desired replicas",
+            "refId": "B"
+            },
+            {
+            "exemplar": true,
+            "expr": "capi_machinedeployment_status_replicas{exported_namespace=~\"$namespace\", machinedeployment=~\"$cluster.*\"}",
+            "hide": false,
+            "interval": "",
+            "legendFormat": "{{ "{{" }}machinedeployment{{ "}}" }}: actual replicas",
+            "refId": "C"
+            }
+        ],
+        "title": "Machinedeployment Status",
+        "type": "timeseries"
+        }
+    ],
+    "refresh": "",
+    "schemaVersion": 31,
+    "style": "dark",
+    "tags": [
+        "CAPI"
+    ],
+    "templating": {
+        "list": [
+        {
+            "allValue": null,
+            "current": {
+            "selected": false,
+            "text": "vsphere-refsa1-bn",
+            "value": "vsphere-refsa1-bn"
+            },
+            "datasource": null,
+            "definition": "label_values(capi_cluster_labels, exported_namespace)",
+            "description": null,
+            "error": null,
+            "hide": 0,
+            "includeAll": false,
+            "label": null,
+            "multi": false,
+            "name": "namespace",
+            "options": [],
+            "query": {
+            "query": "label_values(capi_cluster_labels, exported_namespace)",
+            "refId": "StandardVariableQuery"
+            },
+            "refresh": 1,
+            "regex": "",
+            "skipUrlSync": false,
+            "sort": 0,
+            "type": "query"
+        },
+        {
+            "allValue": null,
+            "current": {
+            "selected": false,
+            "text": "a4-1",
+            "value": "a4-1"
+            },
+            "datasource": null,
+            "definition": "label_values(capi_cluster_labels{exported_namespace=~\"$namespace\"}, cluster)",
```

Comment:
these queries do not match for everyone. Should be `capi_kubeadmcontrolplane_labels`?!

Reply:
> these queries do not match for everyone

I am not sure I fully understand what you mean by that :thinking: 

This query is only for getting the list of the clusters and use it for the `cluster` variable. We could also get the same result by using the query below as the kubeadmControlPlane has the same name as the cluster that owns it:
```
label_values(kubeadmcontrolplane_cluster_labels{exported_namespace=~\"$namespace\"}, kubeadmcontrolplane)
```

Reply:
Some clusters might not use `KubeadmControlPlane` but instead some other ControlPlane provider such as `Talos`.

Reply:
True, `KubeadmControlPlane` is not option here.

Reply:
Ah yes, sorry looks like I misread something here. 

However, let's discuss the dashboards in a separate PRs.